helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -372,6 +381,11 @@ private void refreshBaseline(ResourceControllerDataProvider clusterData,
       Set<String> activeNodes, final CurrentStateOutput currentStateOutput)
       throws HelixRebalanceException {
     LOG.info("Start calculating the new best possible assignment.");
+    CountMetric partialRebalanceCounter = _metricCollector.getMetric(
+        WagedRebalancerMetricCollector.WagedRebalancerMetricNames.PartialRebalanceCounter.name(),
+        CountMetric.class);
+    partialRebalanceCounter.increaseCount(REBALANCE_COUNTER_INCREMENT);
```

Comment:
1L would be acceptable as it would be more readable. Reader knows exactly how much it's getting incremented by.